presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -256,6 +257,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Maximum sample size of the partitions column statistics",
                         hiveClientConfig.getPartitionStatisticsSampleSize(),
                         false),
+                booleanProperty(
+                        IGNORE_CORRUPTED_STATISTICS,
+                        "Experimental: Ignore corrupted statistics",
```

Comment:
I'm not sure we need say experimental here, as we expect this will be a permanent feature. Though I will defer to you as you have more context.